src/main/scala/com/fulcrumgenomics/umi/GroupReadsByUmi.scala

```diff
@@ -514,6 +514,20 @@ class GroupReadsByUmi
     }
   }
 
+  /** The error message when reads are out of order. */
+  private def getReadsOutOfOrderErrorMessage(r1: SamRecord, r2: SamRecord): String = {
+    val errorMessage = if (this.rawTag == "MI") {
+      s"Using 'MI' tag as the 'raw tag', are you sure this is correct?  Did you mean the 'RX' tag?"
+    }
+    else if (this.rawTag == this.assignTag) {
```

Comment:
This should, I think, be a validation error at startup.  I don't think we should allow this.

Reply:
Done